bin/version-packages.sh

```diff
@@ -40,6 +40,7 @@ done
 CURRENT_DIR=$( pwd )
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 JETPACK_ROOT="$(dirname "$SCRIPT_DIR")"
+COMPOSER_VER="$( composer --version | cut -d" " -f3 )"
```

Comment:
It looks like this isn't reliable.

Composer itself might generate the version string, but only for snapshot releases. In that case it will look like you're expecting, like `Composer version $VERSION ($DATE)`.

In other cases, it'll delegate version output to symfony/console. And it depends on a wide range of versions of that package, "^2.8.52 || ^3.4.35 || ^4.4 || ^5.0". It looks like the phar you can download directly from getcomposer.org embeds a 2.x version of symfony/console, which again formats the output like `Composer version $VERSION $DATE`. But if you install in other ways that use a version of symfony/console that includes https://github.com/symfony/console/commit/324d1c1fbdb61cec5420cb2779b50d46099dfdc0 (i.e. 3.2.0+), it'll look like `Composer $VERSION $DATE` instead and this will pull out the date rather than the version.

Reply:
Thanks for that. Trying a different approach in d68c07ba1 .

Reply:
Removed completely in 78baf0f4d